pkg/apis/sedna/v1alpha1/objecttrackingservice_types.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2021 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:shortName=ot
+// +kubebuilder:subresource:status
+
+// ObjectTrackingService describes the data that a objecttrackingservice resource should have
+type ObjectTrackingService struct {
+	metav1.TypeMeta `json:",inline"`
+
+	metav1.ObjectMeta `json:"metadata"`
+
+	Spec   ObjectTrackingServiceSpec   `json:"spec"`
+	Status ObjectTrackingServiceStatus `json:"status,omitempty"`
+}
+
+// ObjectTrackingServiceSpec is a description of a objecttrackingservice
+type ObjectTrackingServiceSpec struct {
+	TrackingWorkers []TrackingWorker `json:"trackingWorkers"`
+	ReidWorkers ReidWorkers `json:"reidWorkers"` 
+}
+
+// TrackingWorker describes the data a tracking worker should have
+type TrackingWorker struct {
```

Comment:
TrackingWorker is redefinited here would lead to compile error, you can place the same definitions into common_types.go